CodingChallenges/CC_125_Fourier_Series/Processing/CC_125_Fourier_Series/CC_125_Fourier_Series.pde

```diff
@@ -0,0 +1,61 @@
+// Fourier Series
+// Daniel Shiffman
+// https://thecodingtrain.com/CodingChallenges/125-fourier-series.html
+// https://youtu.be/Mm2eYfj0SgA
+// https://editor.p5js.org/codingtrain/sketches/SJ02W1OgV
+
+float time = 0;
+ArrayList<Float> wave;
```

Comment:
Can you use the Processing FloatList instead? https://processing.org/reference/FloatList.html

Reply:
Done.